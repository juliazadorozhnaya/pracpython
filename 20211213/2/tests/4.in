[840, 616, 595, 966, 504, 609, 357, 266, 14, 910, 616, 126, 987, 875, 959, 280, 994, 140, 819, 994, 490, 196, 0, 301, 217, 560, 770, 763, 602, 581, 637, 728, 777, 70, 735, 595, 252, 805, 161, 504, 525, 259, 588, 973, 141]
