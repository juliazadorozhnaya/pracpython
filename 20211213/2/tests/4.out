[0, 14, 70, 126, 140, 141, 161, 196, 217, 252, 259, 266, 280, 301, 357, 490, 504, 504, 525, 560, 581, 588, 595, 595, 602, 609, 616, 616, 637, 728, 735, 763, 770, 777, 805, 819, 840, 875, 910, 959, 966, 973, 987, 994, 994]
